### ISSUE TITLE
'KerasTensor' object has no attribute 'input'

### ISSUE THREAD

**Issue by alimynn** [majianjia/nnom#137]
I have a dense network with a InputLayer as the first layer followed by Dense layers and ReLU layers. At the end, there is a Activations layer with sigmoid activation function as can be seen in my screenshot.

![image](https://user-images.githubusercontent.com/9980462/126032196-de58d1cc-e24d-4b9e-8fba-4e39a2a84380.png)

I have tensorflow version 2.5.0 installed and when executing generate_model, i get the error "'KerasTensor' object has no attribute 'input'" in file nnom.py in line 827. Is this an error with the tensorflow version, a bug in the library or did i define anything wrong?

Thanks

P.S.: This is the stack trace and the model definition.
![image](https://user-images.githubusercontent.com/9980462/126032289-1db1c50c-97b9-4a3a-a754-384b2e129b4a.png)

![image](https://user-images.githubusercontent.com/9980462/126032299-488a2b86-43dc-4821-9af9-3d34c66d7c55.png)

**Comment by majianjia** [comment]
Hi @alimynn 
Please try to use `layers.Input` instead of `layers.InputLayer` and see if it helps. 
Other seems fine from me. 
If it doesn't help, please could you let me know your TF version? I recommend using the version above 2.1. I normally test with 2.2 and 2.3
Thanks,

**Comment by alimynn** [comment]
Hi and thank you for your fast answer. I have TF version 2.5 installed first and tested now also with version 2.2. With 2.2 the error comes in line 825 as you can see in the screenshot below.

![image](https://user-images.githubusercontent.com/9980462/126047951-72fed836-bc9b-464f-b244-e4a2673f32cb.png)

Also, changing from layers.InputLayer to layers.Input makes no difference.
What i found now is that when i change the model definition like in the next screenshot, everything works fine.

![image](https://user-images.githubusercontent.com/9980462/126047994-60546642-199e-4607-9d19-1e2642b917fd.png)

**Comment by majianjia** [comment]
Thanks, 
It looks like there is an unsupported layer with the sequential model.
